inc/init.php

```diff
@@ -3,6 +3,20 @@
  * Initialize some defaults needed for DokuWiki
  */
 
+if (!function_exists('isWindows')) {
+    // checks if it is windows OS
+    function isWindows() {
+        return (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') ? true : false;
+    }
+}
+
+if (!function_exists('w2u')) {
+    // convert windows path to unix-like on windows OS
+    function w2u($filename) {
+        return isWindows() ? str_replace('\\', '/', $filename) : $filename;
+    }
```

Comment:
I am not sure this is needed/useful outside of tests. PHP does not care for forward or backward slashes when accessing files. So this is only ever needed when comparing files read from the file system with predefines (as done in the tests). So if this only needed for testing it could be moved to the test's utility functions.